recipe/0001-Fix-pybind11-cmake-path.patch

```diff
@@ -0,0 +1,15 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -91,7 +91,7 @@
+   if(IS_CONDA_BUILD)
+-      include("${CONDA_PREFIX}/share/cmake/pybind11/pybind11Targets.cmake")
+-      include("${CONDA_PREFIX}/share/cmake/pybind11/FindPythonLibsNew.cmake")
+-      include("${CONDA_PREFIX}/share/cmake/pybind11/pybind11Config.cmake")
+-      include("${CONDA_PREFIX}/share/cmake/pybind11/pybind11ConfigVersion.cmake")
+-      include("${CONDA_PREFIX}/share/cmake/pybind11/pybind11Tools.cmake")
++      include("${CMAKE_INSTALL_PREFIX}/share/cmake/pybind11/pybind11Targets.cmake")
```

Comment:
@gagelarsen, this is due to
- https://github.com/conda-forge/pybind11-feedstock/issues/30
- https://github.com/conda-forge/pybind11-feedstock/pull/32

I don't know enough of CMake to gauge what is most appropriate to do here, which is why I didn't open an upstream PR.
This patch is most likely not the "correct" way, but a quick and minimal fix that worked first try.